### ISSUE TITLE
No more Apple remote support ?

### ISSUE THREAD

**Issue by kh** [iina/iina#2723]
**System and IINA version:**

- macOS 10.11.6
- IINA 1.0.6

**Expected behavior:**
In the preferences (keyboard shortcuts) there was a setting to enable Apple remote support. You enabled it and your apple remote worked.

**Actual behavior:**
There is no setting to enable the Apple remote, and the remote doesn't work anymore with iina 1.0.6.

**Comment by ztarbugz** [comment]
There was a PR on mpv which removes this in past: https://github.com/mpv-player/mpv/pull/7205

**Comment by saagarjha** [comment]
Yeah, mpv has removed this because apparently the API doesn't work anymore. There's probably some way to hook up your remote to work with the MediaPlayer API, but I'm not completely sure how you'd go around doing that.

**Comment by megadr01d** [comment]
So we're stuck back on IINA 1.0.4. :/

**Comment by saagarjha** [comment]
Maybe not, if we end up adding AppleScript support.

**Comment by megadr01d** [comment]
Hope you aim for it soon, because we can't even benefit from the recent "exit from legacy fullscreen crash" and possibly other fixes. These fixes should have been on 1.0.5 and breaking changes such as updating mpv should be on a separate build like 1.0.6.

**Comment by low-batt** [comment]
I'm reviewing upstream issues and noticed this one.

I have the [3rd generation Siri Remote](https://support.apple.com/en-us/HT205329) connected to my MacBook Pro running macOS Ventura 13.2.1. As you can see here macOS doesn't show it as an Apple TV Remote, only a "generic" device:
![issue-2723-remote](https://user-images.githubusercontent.com/86170219/227652273-66e16820-a882-4ce8-97fa-b175d94c7b42.jpg)

Bummer.

To control what the remote does I'm using [BetterTouchTool](https://folivora.ai/) which supports the [Siri Remote](https://docs.folivora.ai/docs/901_siri_remote.html):
![issue-2723-config](https://user-images.githubusercontent.com/86170219/227652320-5068f6ec-43e5-4cf2-89eb-a1eb31c96253.jpg)

I've not tuned that configuration. Was just doing some experiments to confirm it can be used to control IINA. Seems to be working.

**Comment by menahishayan** [comment]
Not a fix, but I've found that using a mouse with horizontal scroll works better than most app-based solutions. 
<img width="932" alt="Screenshot 2023-12-24 at 1 59 42 PM" src="https://github.com/iina/iina/assets/8170437/01ef4ba3-573a-41cf-8850-ea7d3a43ece7">

**Bonus Tip:** Get a mouse with a few extra buttons on the side and map them to keyboard keys via [Karabiner](https://karabiner-elements.pqrs.org) (Also free and open source!)
